tests/wpt/mozilla/tests/css/text_overflow_a.html

```diff
@@ -6,15 +6,20 @@
 p {
     width: 128px;
     background: gold;
+}
+.overflow-ellipsis {
+    overflow: hidden;
     text-overflow: ellipsis;
 }
-#hideme {
+.overflow-string {
     overflow: hidden;
+    text-overflow: " [..]";
 }
 </style>
 </head>
 <body>
-<p id=hideme>mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm</p>
+<p class="overflow-ellipsis">mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm</p>
+<p class="overflow-string">mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm</p>
```

Comment:
With your changes, this tests nothing right now (it's a reftest that tests for "mismatch").

You should restore the original test and add a new one that tests for string.